CMakeLists.txt

```diff
@@ -39,3 +41,10 @@ endif()
 if (mmtf_build_examples)
     add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/examples)
 endif()
+
+if (mmtf_install_system)
```

Comment:
why even have an option for this? Nothing will happen unless you do `make install`. I suggest to remove this condition.